OracleUnifiedDirectory/imagetool/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,306 @@
+        files/startOUDInstance.sh \
+        ${SCRIPT_DIR}/
+    RUN chmod a+xr ${SCRIPT_DIR}/* && \
+        chown -R oracle:oracle ${SCRIPT_DIR}
+    
+    USER oracle
+    # Disabling Enpoint Identification for selected CLIs to allow connecting to OUD Instance with any hostname \
+    RUN echo "" >> ${ORACLE_HOME}/oud/config/java.properties && \
+        echo "dsconfig.java-args=-client -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+        echo "dsreplication.java-args=-client -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+        echo "uninstall.java-args=-client -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+        echo "status.java-args=-client -Dcom.sun.jndi.ldap.object.disableEndpointIdentification=true" >> ${ORACLE_HOME}/oud/config/java.properties && \
+        ${ORACLE_HOME}/oud/bin/dsjavaproperties
+    # run container health check
+    HEALTHCHECK --interval=1m --start-period=5m \
+        CMD "${SCRIPT_DIR}/checkOUDInstance.sh" > /dev/null || exit 1
+    
+    # Create and Start OUD Instance
+    # ENTRYPOINT ["sh", "-c", "${SCRIPT_DIR}/createAndStartOUDInstance.sh"]
+    CMD ["sh", "-c", "${SCRIPT_DIR}/createAndStartOUDInstance.sh"]
+    
+# 8. Create OUD image
+
+Now we can build the OUD image with the imagetool. 
+
+The following parameters are provided as input to the create command,
+
+* jdkVersion - JDK version to be used in the image.
+* type - type of image to be built.
+* version - version of the image.
+* tag - tag name for the image.
+* additionalBuildCommands - additional build commands provided as a text file.
+* addtionalBuildFiles - path of additional build files as comma separated list.
+
+
+Below a sample command used to build OUD image,
+
+    $ imagetool create --jdkVersion=8u241 --type oud --version=12.2.1.4.0 \
+        --tag=oud-with-patch:12.2.1.4.0 \
+        --additionalBuildCommands <Path to Repo directory OracleUnifiedDirectory>/imagetool/12.2.1.4.0/additionalBuildCmds.txt \
+        --additionalBuildFiles \
+            <Path to Repo directory OracleUnifiedDirectory>/dockerfiles/12.2.1.4.0/container-scripts \
+        --patches 30851280
+
+# 9. Sample Dockerfile genered with imagetool
+Below is the content of sample Dockerfile generated by imagetool in dryrun for OUD image,
+
+	########## BEGIN DOCKERFILE ##########
+	#
+	# Copyright (c) 2019, 2020, Oracle and/or its affiliates.  All rights reserved.
+	#
+	# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+	#
+	#
+	FROM oraclelinux:7-slim as OS_UPDATE
+	LABEL com.oracle.weblogic.imagetool.buildid="f3557b16-8e34-4bf0-be6a-baa06fd8c137"
+	USER root
+	
+	RUN yum -y --downloaddir= install gzip tar unzip \
+	 && yum -y --downloaddir= clean all \
+	 && rm -rf /var/cache/yum/* \
+	 && rm -rf
```

Comment:
This line seems particularly dangerous. Why is there a random `rm -rf` with no target?